### PR TITLE
add router(v6) with placeholder pages

### PR DESCRIPTION
**Pull request by ulositska**
For now I went with an older version, LMN if I should change it to the newer one

### CODE REVIEW

File: package-lock.json
Comment (okosohor) [inline package-lock.json:1]: add only ./src

File: package.json
Comment (okosohor) [inline package.json:1]: add only ./src

File: src/App.tsx [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
-export const App: React.FC = () => <h1>Hello!</h1>;
+export const App: React.FC = () => (
+  <div className={'section'}>
+    {/* <Header /> */}
+    <div className="container">
+      <Routes>
+        {/* <Route path="*" element={<PageNotFound />} /> */}
+        <Route path="/" element={<Home />} />
+        <Route path="/home" element={<Navigate to="/" replace />} />
+        <Route path="/phones" element={<Phones />} />
```
Comment (okosohor) [inline src/App.tsx:16]: need to add cart, tablets etc.

File: src/App.tsx [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
 import React from 'react';
+import { Routes, Route, Navigate } from 'react-router-dom';
 import './App.scss';
+import { Phones } from './Pages/Phones/Phones';
+import { Home } from './Pages/Home/Home';
 
-export const App: React.FC = () => <h1>Hello!</h1>;
+export const App: React.FC = () => (
+  <div className={'section'}>
+    {/* <Header /> */}
```
Comment (okosohor) [inline src/App.tsx:10]: i think mb we can use outlet

File: src/index.tsx [hunk 3]
```diff
@@ -8,6 +9,8 @@ const root = ReactDOM.createRoot(
 );
 root.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
```
Comment (okosohor) [inline src/index.tsx:12]: for GH need hashRouter

**Review by okosohor — CHANGES_REQUESTED** [review]

**Comment by Ihor-Karpyn** [comment]
Looks like u use a different node version, it's the reason for the huge page-lock.

File: package-lock.json
Comment (Ihor-Karpyn) [reply package-lock.json:1]: package-lock should be added to this pull request

**Review by annholovko — CHANGES_REQUESTED** [review]
add routes for tablets, accessories, cart, favourites, cart etc, and add this pages too.
see https://docs.google.com/document/d/14pL86Alwv6a1gw7iq2Qn89MiKTwFWNIfsYllQmfLfR8/edit

**Pull request closed without merging**